examples/jsm/nodes/accessors/MorphNode.js

```diff
@@ -19,12 +19,12 @@ class MorphNode extends Node {
 
 	}
 
-	setupAttribute( builder, name, assignNode = positionLocal ) {
+	setupAttribute( name, assignNode = positionLocal ) {
 
 		const mesh = this.mesh;
 		const attributes = mesh.geometry.morphAttributes[ name ];
 
-		builder.stack.assign( assignNode, assignNode.mul( this.morphBaseInfluence ) );
+		assignNode.assign( assignNode.mul( this.morphBaseInfluence ) );
```

Comment:
Would be great to also update instances of code like this to use `mulAssign`, `addAssign`, etc (but this can be done in a separate PR).